src/main/java/org/sonatype/repository/conan/internal/hosted/ConanHostedRecipe.groovy

```diff
@@ -384,6 +385,15 @@ class ConanHostedRecipe
     )
   }
 
+  static Builder downloadConanExportZip() {
```

Comment:
```suggestion
  static Builder downloadConanExportZip(final String url) {
```

Being that this is virtually identical to downloadConanPackageZip, you could consolidate this Builder with the other by accepting a String with the URL and using that in the TokenMatcher